app/views/schools/register_ect_wizard/previous_ect_details.html.erb

```diff
@@ -0,0 +1,47 @@
+
+<% page_data(
+    title: "#{@ect.full_name} has been registered before",
+    error: @wizard.current_step.errors.present?,
+    backlink_href: @wizard.previous_step_path
+) %>
+
+<%= govuk_summary_list(card: { title: 'Previously registered details' } ) do |list| %>
+  <% list.with_row do |row| %>
+    <%= row.with_key { "School name" } %>
+    <%= row.with_value { Teachers::CurrentSchools.new(trn: @ect.trn).ect_at&.name || 'Not confirmed' } %>
+  <% end %>
+
+  <% list.with_row do |row| %>
+    <%= row.with_key { "Induction start date" } %>
+    <%= row.with_value { Teachers::PreviousECTRegistrationDetails.new(trn: @ect.trn).induction_start_date&.to_fs(:govuk) || 'Not confirmed' } %>
```

Comment:
`<%= row.with_value { @ect.induction_start_date&.to_fs(:govuk) || 'Not confirmed' } %>`

Reply:
and the rest of data in this view

Reply:
Thank you 👍🏽 

refactored here: https://github.com/DFE-Digital/register-early-career-teachers-public/pull/554/commits/780b886865b761e634b69de8705b31655d730d6f